zebra-rpc/src/server.rs

```diff
@@ -209,7 +222,7 @@ impl RpcServer {
             .layer_fn(FixRpcResponseMiddleware::new);
 
         let server_instance = Server::builder()
-            .http_only()
+            // .http_only()
```

Comment:
Uncommenting this doesn't seem to make a major difference, is there something more subtle I'm not noticing?

Reply:
I think Andrew was trying to get streaming to work. This will be redundant if we are in fact moving to grpc.